src/client/clouds.cpp

```diff
@@ -322,9 +327,7 @@ void Clouds::updateMesh()
 				break;
 			}
 
-			v3f pos(p0.X, m_params.height * BS, p0.Y);
-
-			for (video::S3DVertex &vertex : v) {
+			for (video::S3DVertex& vertex : v) {
```

Comment:
Rebase issue. Would you please be so nice to clean these changes? (from what I can see, this only affects `clouds.cpp`)